src/utils/getClipboardImage.ts

```diff
@@ -46,7 +46,10 @@ const getClipboardImage = (ctx: PicGo): Promise<ClipboardImage> => {
         '-nologo',
         '-sta',
         '-executionpolicy', 'unrestricted',
-        '-windowstyle', 'hidden',
+        // '-windowstyle', 'hidden',
+        // prevent powershell console from closing after running script 
+        // https://github.com/PicGo/PicGo-Core/issues/32
+        '-noexit',
```

Comment:
![image](https://user-images.githubusercontent.com/12621342/76673214-105e9300-65de-11ea-93cb-3fc8fc85f24b.png)
please add `execution.kill()` in this line to kill the process after get the result.

Reply:
And please use [picgo/convention](https://github.com/PicGo/bump-version#picgo-convention) to submit the commit message just like the commit before.

Reply:
Thanks for pointing out. Also I refer to this Stack Overflow problem and find a similar solution from Joe Johnston's comment. 
The main function inside of windows10.ps1 does not kill process after generating clipboard image to $imagePath. 
[How to exit PowerShell.exe inside of CMD](https://stackoverflow.com/questions/42774454/how-to-exit-powershell-exe-inside-of-cmd)